.pre-commit-config.yaml

```
@@ -4,11 +4,22 @@ repos:
   hooks:
     - id: terraform_fmt
     - id: terraform_tflint
+      name: Validate with tflint the uw hosted team configs
       args:
-        - --args=--config=__GIT_WORKING_DIR__/.tflint.hcl
+        - --args=--config=__GIT_WORKING_DIR__/.tflint-uw-team-hosted.hcl
         # use compact, so that errors will be reported by the GH action see https://github.com/terraform-linters/setup-tflint?tab=readme-ov-file#checks
         - --args=--format=compact
+        # Using this allows issues to appear directly in the PR code. See https://github.com/antonbabenko/pre-commit-terraform?tab=readme-ov-file#terraform_tflint
         - --hook-config=--delegate-chdir
+      exclude: "^.*/kafka-shared-msk/.*$"
+    - id: terraform_tflint
+      name: Validate with tflint the MSK team configs
+      args:
+        - --args=--config=__GIT_WORKING_DIR__/.tflint-msk.hcl
+        # use compact, so that errors will be reported by the GH action see https://github.com/terraform-linters/setup-tflint?tab=readme-ov-file#checks
+        - --args=--format=compact
+        # We won't be able to use --hook-config=--delegate-chdir, as we need to determine the current folder in our plugin tflint-ruleset-kafka-config.
+      files: "^.*/kafka-shared-msk/.+/.*$"
```

To keep things simple, we could probably just match the start of the path (or is there a reason we enforce the depth with the `/.+/.` bit?)

```suggestion
      files: "^.*/kafka-shared-msk/"
```

---

Yea, as we don't wanna run it on the folder dev-asw/kafka-shared-msk/ as it doesn't contain any module. Only on its subfolders